snapcraft/plugins/catkin.py

```diff
@@ -889,9 +889,11 @@ def find(self, package_name):
 
     def _run(self, arguments):
         with tempfile.NamedTemporaryFile(mode='w+') as f:
-            lines = ['export PYTHONPATH={}'.format(os.path.join(
-                self._catkin_install_path, 'usr', 'lib', 'python2.7',
-                'dist-packages'))]
+            lines = ['export PYTHONPATH={}:{}'.format(
+                os.path.join(self._catkin_install_path, 'usr', 'lib',
+                             'python2.7', 'dist-packages'),
+                os.path.join(self._catkin_install_path, 'opt', 'ros',
+                             'kinetic', 'lib', 'python2.7', 'dist-packages'))]
```

Comment:
I may have jumped the gun a little. While this change does fix the error I was seeing, it's still not finding the underlay packages, which I think is because the _setup_util.py in my underlay is still pointing to the directory it was dumped from. It would make sense that if that were properly sourced it would find the catkin package. I'll report back once I get everything working.

I submitted the CLA just before making the pull request.